packages/monaco/tsconfig.json

```diff
@@ -4,4 +4,7 @@
 	"references": [
 		{ "path": "../language-service/tsconfig.json" },
 	],
-}
\ No newline at end of file
+	"compilerOptions": {
+		"esModuleInterop": true
```

Comment:
The monaco package does not use `import default` syntax, is this still needed?

Reply:
Currently `@volar/monaco` doesn’t use `import default`, but it may need to due to future dependencies. This shouldn’t ever be a problem for ESM, as dependants should also have `esModuleInterop` enabled.

The VS Code team recommends enabling `verbatimModuleSyntax` over disabling `esModuleInterop` (though this has more implications).